src/main/steps/respondent/legal-jurisdiction-of-the-courts/content.ts

```diff
@@ -34,18 +34,128 @@ const en = ({ isDivorce, partner, required, userCase, isJointApplication }: Comm
       },
       inWhichCountryIsYourLifeMainlyBased: {
-        required: 'You need to tell us which country your life is mainly based',
+        required: 'You need to enter which country your life is mainly based.',
       },
     },
+    whatThisMeansJurisdiction: {
+      heading1: 'Habitual residence',
+      line1:
+        'If your lives are mainly based in England or Wales then you’re what is legally known as ‘habitually ' +
+        'resident’.',
+      line2:
+        'This may include working, owning property, having children in school, and your main family life taking ' +
+        'place in England or Wales.',
+      line3:
+        'The examples above are not a complete list of what makes up habitual residence. Just because some of them' +
+        ' apply to you, that does not mean you’re habitually resident. If you’re not sure, you should get legal advice.',
+      heading2: 'Domicile',
+      line4:
+        'Your domicile is usually the place in which you were born, regard as your permanent home and to which ' +
+        'you have the closest ties.',
+      line5:
+        'However, domicile can be more complex. For example, if you or your parents have moved countries in the past.',
+      line6: 'When you’re born, you acquire the ‘domicile of origin’. This is usually:',
+      bulletPoint1: 'the country your father was domiciled in if your parents were married',
+      bulletPoint2:
+        'the country your mother was domiciled in if your parents were unmarried, ' +
+        'or your father had died before you were born',
+      line7:
+        'If you leave your domicile of origin and settle in another country as an adult, the new country may ' +
+        'become your ‘domicile of choice’.',
+      line8: 'If you’re not sure about your domicile, you should get legal advice.',
+      heading3: 'Residual jurisdiction',
+      line9: `Usually, to be eligible for residual jurisdiction you or your ${partner} must be domiciled in England.
+      Neither of you must be nationals of or habitually resident in, another country in the EU (except Denmark).`,
+      line10:
+        'In addition, if you’re married to a member of the same sex, you may be eligable for residul ' +
+        'jurisdiction if: (all the following apply):',
+      bulletPoint3: 'you married each other in the UK',
+      bulletPoint4:
+        'neither of you are nationals of, or habitually resident in, another country in the EU (except Denmark)',
+      bulletPoint5: `it would be in the interests of natural justice for the court to consider this application
+        (this may apply if, for example, your home country does not allow ${
+          isDivorce ? 'divorce' : 'ending of a civil partnership'
+        }  between same-sex couples)`,
+      line11:
+        'However, residual jurisdiction can be complex. If you’re not sure whether this applies to you then you should get legal advice',
+    },
```

Comment:
this content already exists in src/main/app/jurisdiction/moreDetailsContent.ts
it's easier to import it from there and use those instead of adding them just for this page, as it's used in quite a few different places. You can also add the Welsh translations for it there too

Reply:
Also the residual jurisdiction text is found in src/main/steps/applicant1/review-your-application/content.ts 
maybe you could also copy that out into the moreDetailsContent for consistency

Reply:
I just checked, and for this page it's the jurisdictionMoreDetailsContent function in moreDetailsContent.ts that'll need updating to be used for Welsh translations too

Reply:
Thanks Lara, I'll work on those comments.

Reply:
Morning Lara, I made some changes based off your comments, let me know if there are some more edits to be made.